WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -52,6 +52,15 @@ final class ProductSelectorViewModel: ObservableObject {
     ///
     @Published private(set) var productRows: [ProductRowViewModel] = []
 
+    /// Determines if multiple item selection is supported, `false` by default.
+    /// Mainly used to allow multi-selection on the Order Creation flow
+    ///
+    let supportsMultipleSelections: Bool
+
+    /// Determines if is possible to toggle all variatoon items upon selection
+    ///
+    let enableTogglingSelectionForAllVariations: Bool
```

Comment:
I think the comment pattern is more meaningful for this property naming:

`let toggleAllVariationsOnSelection`

Reply:
> comment pattern

Interesting! Definitely more readable, changed on https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/e1142d11405dbc38ed7317ea85d6538e5842c088